File: 3379-score-of-a-string/3379-score-of-a-string.js
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function(s) {
      let n=s.length;
        let result =0;
        for(let i=1; i<n; i++){
            result +=Math.abs(s.charCodeAt(i-1)-s.charCodeAt(i));
        }
        return result;
};

File: 3379-score-of-a-string/3379-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
      int n = s.length();
    int result = 0;
    for (int i = 1; i < n; i++) {
        result += Math.abs(s.charAt(i) - s.charAt(i - 1));
    }
    return result;
    }
}